clarifai/models/model_serving/model_config/triton_config.py

```diff
@@ -136,60 +155,64 @@ def __post_init__(self):
     Set supported input dims and data_types for
     a given model_type.
     """
-    image_input = InputConfig(name="image", data_type="TYPE_UINT8", dims=[-1, -1, 3])
-    text_input = InputConfig(name="text", data_type="TYPE_STRING", dims=[1])
+    image_dims = self.image_shape
+    image_dims.append(3)  # add channel dim
+    image_input = InputConfig(name="image", data_type=DType.TYPE_UINT8, dims=image_dims)
+    text_input = InputConfig(name="text", data_type=DType.TYPE_STRING, dims=[1])
+    # del image_shape as it's a temporary config that's not used by triton
+    del self.image_shape
```

Comment:
I'm confused why this needs to be deleted?

Reply:
because it's not a valid triton config: just using it to capture shapes to update the input dims which are later serialized.